internal/api/vm/converters.go

```diff
@@ -48,6 +48,23 @@ func ToServiceOfferingsVM(offerings *types.ServiceOfferings) ServiceOfferings {
 	return toReturn
 }
 
+func ToServiceInstancesVM(instances *types.ServiceInstances) ServiceInstances {
```

Comment:
Is this function somehow tested using unit test approach?
Testing if NumItems == len(Items) for corner cases?

Reply:
Not changing NumItems in the pr after its rebase.